inc/macro_utils/macro_utils.h

```diff
@@ -238,20 +236,6 @@ typedef struct ENUM_VALUE_AND_STRING_TAG
 }ENUM_VALUE_AND_STRING;
 
 #define MU_DEFINE_ENUM_VALUE_AND_STRING(enumerationConstant, constantExpression) {enumerationConstant, MU_TOSTRING(enumerationConstant)},
```

Comment:
looks like it's not used anymore

Reply:
Love it, removed!

---
In reply to: [636327543](https://github.com/Azure/macro-utils-c/pull/75#discussion_r636327543) [](ancestors = 636327543)